input/pagecontent/index.md

```diff
@@ -81,4 +84,9 @@ Our thanks to the contributors to the CodeX/GenomeX data exchange workstream whi
 * James Patterson - MITRE
 * Vasil Petyev - Epic
 * Brian Reuter - Tempus
-* May Terry - MITRE
\ No newline at end of file
+* May Terry - MITRE
+
+<!-- LINKS -->
+
+[here]:
+pagecontent/transport.md
```

Comment:
Is this supposed to be a hyperlink? The syntax would be [name shown in text](hyperlink).

You can start with the html file and no need to put 'pagecontent', which is only used as the directory structure by sushi.

For example, you can say [GenomeX Tranport](transport.html).